cadence/scripts/nftStorefront/get_listing_item.cdc

```diff
@@ -1,36 +1,92 @@
 import NonFungibleToken from "../../contracts/NonFungibleToken.cdc"
+import MetadataViews from "../../contracts/MetadataViews.cdc"
 import NFTStorefront from "../../contracts/NFTStorefront.cdc"
 import KittyItems from "../../contracts/KittyItems.cdc"
 
-pub struct SaleItem {
+pub struct ListingItem {
```

Comment:
Again, it feels like this should be something that is defined in the KittyItems contract or storefront or something like that